### ISSUE TITLE
User input from Single-line Text field not being sanitised in e-mail notification

### ISSUE THREAD

**Issue by rickbergmans** [verbb/formie#1226]
### Describe the bug

User input for "Single-line Text" fields is not being sanitised when using {fieldName}, allowing HTML injection. When using {htmlContent}, the field is escaped properly.

### Steps to reproduce

Add a single line text field to any form
Add an email notification with the {fieldName} specified
Submit <a href="https://www.bad-link.test">Very sketchy stuff</a> into a Single-line Text field
Email arrives with a clickable "click here" link (other HTML input also works)

![Scherm­afbeelding 2023-01-03 om 10 28 51](https://user-images.githubusercontent.com/17927472/210333513-15b07e70-81d6-41c5-b56a-fdbde9017606.png)

### Form settings

Notification uses {fieldName} instead of {htmlContent}

### Craft CMS version

3.7.62

### Plugin version

1.5.3

### Multi-site?

No

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.6.21",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.6.22](https://github.com/verbb/formie/releases/tag/1.6.22)
